qa/rpc-tests/test_framework/util.py

```diff
@@ -67,6 +67,9 @@ def check_json_precision():
     if satoshis != 2000000000000003:
         raise RuntimeError("JSON encode/decode loses precision")
 
+def count_bytes(hex_string):
+    return len(bytearray.fromhex(hex_string))
```

Comment:
I considered that nit, but right now it checks it is a valid hex string, which is kind of useful for a test